File: 1791-RichestCustomerWealth/1791-RichestCustomerWealth.java
// Last updated: 17/07/2025, 11:25:48
class Solution {
    public int maximumWealth(int[][] accounts) {
        int maxWealth = 0;
        for(int i=0; i<accounts.length; i++){
            int sum = 0;
            for(int j = 0; j<accounts[i].length; j++){
                sum += accounts[i][j];
            }
            if(sum > maxWealth){
                maxWealth = sum;
            }
        }
        return maxWealth;
    }
}

File: 1791-RichestCustomerWealth/1791-RichestCustomerWealth.py
# Last updated: 17/07/2025, 11:25:43
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int: 
        richest = sum(accounts[0])
        for i in range(1, len(accounts)):
            if sum(accounts[i]) > richest:
                richest = sum(accounts[i])
        return richest